transcom/mymove
# B 18181 INT deprecate ppm front end code

## [B-18181](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3a871871)

## Summary

Remove PPM related front end code that is no longer used. I was quite thorough in investigating what code to remove, and making sure it does not break the application. All tests pass, but with 6000+ lines of code removed, I could've missed something.

These are to be checked by a reviewer.


### How to test

1. Login as a customer, create a new PPM, follow through the entire process (submit a move, approve with SC, submit move documents, review as closeout) and ensure that the app works as it should
2. Generally test throughout the application as much as possible.
